examples-nosql-node-sdk/sqlexamples/AddData.js

```diff
@@ -282,20 +282,24 @@ const acct3= `
 doAddData();
 
 async function doAddData() {
+   let handle;
    try {
       /* UNCOMMENT the line of code below if you are using Oracle NoSQL Database Cloud service. Leave the line commented if you are using onPremise database
```

Comment:
comment line looks too long, must be under 80 char.   I think it needs to be on multiple lines instead of wrapping.  Use the format below.  Do for all long comments.
/*
*   ddkdkdkdkdkd
*   dkdkdkdkdkkd
* /